sdk/bootstrap/main.go

```diff
@@ -305,6 +314,51 @@ func installDCOSCertIntoJRE() {
 	log.Println("Successfully installed the certificate.")
 }
 
+func decodeKeystores() error {
```

Comment:
1. Please add a test, but no need to wire it up yet
2. Does that plugin support the fact that we're outside of GOPATH? (e.g. that we shoehorn ourselves into it)